alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -17,7 +17,10 @@ use std::io::Error;
 use std::mem;
 use std::os::windows::io::IntoRawHandle;
 use std::ptr;
-use std::sync::Arc;
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
```

Comment:
We don't use nested imports like this in Alacritty, they're usually too complex and this would just be two normal short lines with the more traditional layout.